dc-cudami-admin/src/main/resources/templates/collections/view.html

```diff
@@ -49,6 +49,14 @@ <h1 th:text="#{collection}">Collection</h1>
               <div class="col-md-3"><label class="font-weight-bold" th:text="#{id}">ID</label></div>
               <div class="col-md-9"><span th:text="*{uuid}">b7a245fe-da46-4d7d-a8e4-a7ee8f24f840</span></div>
             </div>
+            <div class="row" th:if="${not #lists.isEmpty(parents)}">
+              <div class="col-md-3"><label class="font-weight-bold" th:text="#{parent_collections}">Parent collections</label></div>
+              <div class="col-md-9">
+                <th:block th:each="parent,iterStat: ${parents}">
```

Comment:
```suggestion
                <th:block th:each="parent: ${parents}">
```
`iterstat` wird ja gar nicht verwendet.